utils/utils.go

```diff
@@ -22,18 +22,12 @@ func OpenInBrowser(url string) error {
 }
 
 func RenderMarkdown(text string) (string, error) {
-	style := "notty"
```

Comment:
With this removal, we lose the feature that, when color is disabled, no coloring styles should take place. I think that `notty` is important to keep unless the output is to a terminal (or color was otherwise forced)

Reply:
Agreed. Will put back.

Reply:
Once the underlying issue in Glamour is resolved, I *think* the new implementation I just pushed should be acceptable for `notty` functionality.

Reply:
Fixed.